eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Visitors/ReNameResourceUpdateModelVisitor.cs

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.TypeSpec.Generator.ClientModel;
+using Microsoft.TypeSpec.Generator.Input;
+using Microsoft.TypeSpec.Generator.Providers;
+using System;
+using System.Collections.Generic;
+
+namespace Azure.Generator.Management.Visitors
+{
+    internal class ReNameResourceUpdateModelVisitor : ScmLibraryVisitor
```

Comment:
since this is about renaming, can we put it in https://github.com/Azure/azure-sdk-for-net/blob/main/eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Visitors/NameVisitor.cs?